docs/portal_config.md

```diff
@@ -388,7 +388,17 @@ Below is an example, with inline comments describing what each JSON block config
     "dropdowns": {} // optional; dropdown groupings for buttons
   },
   "discoveryConfig": { // config for Discovery page. Required if 'featureFlags.discovery' is true. See src/Discovery/DiscoveryConfig.d.ts for Typescript schema.
+    "requireLogin": false, // optional, defaults to false. If true, requires user to sign in before seeing the Discovery page
     "features": {
+      "exportToWorkspaceBETA": { // configures the export to workspace feature. If enabled, the Discovery page data must contain a field which is a list of GUIDs for each study. See `manifestFieldName`
+          "enable": boolean
+          "enableDownloadManifest": boolean // enables a button which allows user to download a manifest file for gen3 client
+          "documentationLinks": {
+              "gen3Client": string // link to documentation about the gen3 client. Used for button tooltips
+              "gen3Workspaces": string // link to documentation about gen3 workspaces. Used for button tooltips.
```

Comment:
are these links subject to change from commons to commons?

The way those logic got wired up in the code implies this whole `documentationLinks` is required, but if they are not that volatile, why not just hard code them as what we have done for the `/submission` page?

Reply:
You're right, I think this is probably not worth the complexity. I'll change these from configurable values to constants declared in the file.